### DISCUSSION TITLE
Issue with loading existing signature, making changes, and then saving

### DISCUSSION THREAD

**Discussion by samslmmr** [szimek/signature_pad#721] [Q&A]
I am able to load a previous signature into the component using

`this.sigPad = new SignaturePad(canvas);`
 `this.sigPad.fromDataURL("datastring");`

My previous signature loads and then I can continue editing / adding to it.  However, when my save button is clicked and I use
`this.sigPad.toSVG();`
or 
`this.sigPad.toDataURL("image/svg+xml")`
I only get the edits I made. I do not get the complete image. 

How do I get the previous signature and the edits?

**Comment by UziTech** [comment]
`fromDataURL` just loads the image onto the canvas. There is no data loaded from the previous signature.

If you want to save a previous signature and reload the data you will need to save it with `toData` and load it back with `fromData`.
